tests/src/runtimeApi/memory/hipMemcpy.cpp

```diff
@@ -280,6 +287,26 @@ void memcpytest2_for_type(size_t numElements) {
     }
 }
 
+#ifdef _WIN32
+void memcpytest2_get_host_memory(size_t& free, size_t& total) {
+    MEMORYSTATUSEX status;
+    status.dwLength = sizeof(status);
+    GlobalMemoryStatusEx(&status);
+    free = status.ullAvailPhys;
+    total = status.ullTotalPhys;
+}
+#else
+struct sysinfo memInfo;
+void memcpytest2_get_host_memory(size_t& free, size_t& total) {
+    sysinfo(&memInfo);
+    long long freePhysMem=memInfo.freeram + memInfo.freeswap;
```

Comment:
why do you add freeswap? we only want to use physical memory right?

Reply:
I thought if we include swap, then machines with low RAM (4GB), but high GPU RAM (32GB) can still run if they have swap space (32GB). Or should we just use physical?